docs/handbook/getting-started.md

```diff
@@ -1532,6 +1530,133 @@ Use the following steps to change the root password:
 * Once changed you can exit the su session
 * You should be able to login/authenticate as root now.
 
+## Updating OpenIndiana /dev to /hipster
+
+This section describes how to update your existing OpenIndiana /dev installation to OpenIndiana /hipster.
+
+Note, that although direct updating from /dev to /hipster can be possible, it's not exhaustively tested.
+
+There are several techniques which you can use to update your systems, including root pool installation from OpenIndiana ISO.
+Complete reinstall also can be a decent option.
+
+Further it's considered that you were warned and decided to do more-or-less direct update from /dev to /hipster.
+
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
```

Comment:
Should there be HTML markup in markdown formatted docs? I think some MD constructs can be used to same effect natively and portably?

Reply:
...And more so for empty tags that bring no visible value ;)

Reply:
It is used throughout the book. It can be fixed globally, but I'm not going to fix this pattern in this PR.